src/Microsoft.Health.Fhir.TemplateManagement/Client/OrasClient.cs

```diff
@@ -24,24 +26,31 @@ public OrasClient(string imageReference)
             _imageReference = imageReference;
         }
 
-        public async Task PullImageAsync(string outputFolder)
+        public async Task<string> PullImageAsync(string outputFolder)
         {
             string command = $"pull  \"{_imageReference}\" -o \"{outputFolder}\"";
-            await OrasExecutionAsync(command, Directory.GetCurrentDirectory());
+            string output = await OrasExecutionAsync(command, Directory.GetCurrentDirectory());
+            Console.WriteLine(output);
+            Regex rx = new Regex("(?<rule>[A-Za-z][A-Za-z0-9]*([+.-_][A-Za-z][A-Za-z0-9]*)*):(?<digest>[0-9a-fA-F]{32,})");
```

Comment:
This regex looks useful as it extracts the digest. We can save it somewhere instead of a local variable.